csrc/keyutils.cpp

```diff
@@ -7,26 +7,27 @@
 #include <openssl/err.h>
 #include <openssl/evp.h>
 #include <openssl/rsa.h>
+#include <iostream>
 
 namespace AmazonCorrettoCryptoProvider {
 
-EVP_PKEY* der2EvpPrivateKey(
-    const unsigned char* der, const int derLen, bool shouldCheckPrivate, const char* javaExceptionClass)
+EVP_PKEY* der2EvpPrivateKey(const unsigned char* der,
+    const int derLen,
+    const int nativeValue,
```

Comment:
`nativeValue` means nothing to me. Can you rename this to something like `evpType`?

Reply:
Updated. `nativeValue` was used in the [pkcs82evp](https://github.com/corretto/amazon-corretto-crypto-provider/blob/d71028c67c2e06cadf44c5eaa138e11d26192cb9/csrc/java_evp_keys.cpp#L96) function, so I copied the naming over to `der2EvpPrivateKey`. Would it be worthwhile to also change the variable name there (pkcs82evp)?

Reply:
That'd be nice, but if it looks like you'd end up changing a lot of unrelated code don't bother.

Reply:
Sounds good, just updated the name to evpType in the native methods and corresponding java methods.